database/seed.js

```diff
@@ -42,6 +33,7 @@ const seedDb = () => {
   });
 };
 
+// seed with 100 products
 let i = 0;
```

Comment:
Why not use a for loop here instead of initializing a global variable + a while loop? I don't see where i is being decremented, would that cause an infinite loop?

Reply:
The condition is that i <100, and I increment within the while loop, so infinite crisis averted!

I could use a for loop, yes. Do you think it's better suited for the job here?

Reply:
I see, I didn't notice the last few lines of your while loop (GitHub hid them from me 🤷‍♂ )

I think as a general rule of thumb it's considered more readable/efficient to use a for loop rather than declaring a global variable and incrementing it within a while loop. They obviously give the same result but that's just my .02 :)

One more thing: Don't forget that last newline at the end of the file 👍